tests/bugfixes/github/test_issue_1959.py

```diff
@@ -15,7 +15,7 @@ def setUp(self):
     filename      = path("$data_path/issue_1959_poc.xmp")
     filename_ref  = path("$data_path/issue_1959_poc.xmp.out")
     
-    commands = ["$exiv2 -Pkvt $filename"]
+    commands = ["TZ=UTC $exiv2 -Pkvt $filename"]
```

Comment:
@piponazo @mallman I seem to have found the existing setting we need to use for this :tada: 


```suggestion
    env = {
        'TZ': 'UTC'
    }
    commands = ["$exiv2 -Pkvt $filename"]
```

Reply:
@mallman if you apply that change and it works, I'll close #2079 .

Reply:
WFM. Let's see if the Windows tests pass...